### PR TITLE
TVB-2901: Solve problems during sprint end testing

### PR DESCRIPTION
**Pull request by VinczeRobert**

### CODE REVIEW

File: framework_tvb/tvb/core/services/simulator_service.py [hunk 1]
```diff
@@ -63,7 +64,7 @@ def __init__(self):
     @staticmethod
     def _reset_model(session_stored_simulator, form):
         session_stored_simulator.model = type(session_stored_simulator.model)()
-        vi_indexes = form.determine_indexes_for_chosen_vars_of_interest(session_stored_simulator)
+        vi_indexes = MonitorForm.determine_indexes_for_chosen_vars_of_interest(session_stored_simulator)
```
Comment (popaula937) [inline framework_tvb/tvb/core/services/simulator_service.py:67]: The `form` argument does not seem to be necessary now

Comment (liadomide) [reply framework_tvb/tvb/core/services/simulator_service.py:67]: True, but now we have an undesired import from adapters in core. Can't we do it differently?

Comment (VinczeRobert) [reply framework_tvb/tvb/core/services/simulator_service.py:67]: I will try a different way then.

Comment (popaula937) [reply framework_tvb/tvb/core/services/simulator_service.py:67]: Right, missed that. 
Looking better at these methods, they don't seem to belong under the `Monitor`.
How about moving them under `SimulatorAdapterModel` which is in `core`?

Comment (popaula937) [reply framework_tvb/tvb/core/services/simulator_service.py:67]: Then you would have: session_stored_simulator.determine_indexes_for_chosen_vars_of_interest(..)

Comment (VinczeRobert) [reply framework_tvb/tvb/core/services/simulator_service.py]: I didn't read your comment before doing in another way, I think this one is maybe better. I thought about putting it in the SimulatorAdapterModel class but I don't really see methods on the ViewModel classes so I thought not to do that.

Comment (popaula937) [reply framework_tvb/tvb/core/services/simulator_service.py]: But this is still an import from adapters to the core and we are trying to avoid that

Comment (VinczeRobert) [reply framework_tvb/tvb/core/services/simulator_service.py]: Oh yeah, that method is in adapters too... I will put it on the ViewModel then, don't have any other ideas.

**Review by popaula937 — APPROVED** [review]

**Pull request merged**